pages/common/laravel.md

```diff
@@ -0,0 +1,15 @@
+# laravel
+
+> The PHP Framework For Web Artisans.
```

Comment:
I find this an interesting description. Although it is the official description for the actual framework, I assume this is the CLI installer (i.e. [`laravel/installer`](https://github.com/laravel/installer)).

Perhaps it might be better to imply that it's a CLI installer?

Maybe something like:
>A command line installer for the Laravel framework.